### PR TITLE
Scoreboard

### PR DESCRIPTION
**Pull request by brianchitester**
hopefully this works.

### CODE REVIEW

**Comment by brianchitester** [comment]
Is there a better way to test changes rather than in production?

File: scripts/points.coffee [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	scoreboard =
+		chris: 10
+		ravi: 1
+		brian: -5
+		tommy: 0
+		christina: 0
+		nick: 0
+		greyson: 0
+
+	robot.hear /studbot give (.*) (\d*) points/i, (msg) ->
```
Comment (TerribleDev) [inline scripts/points.coffee:11]: use .respond and then remove studbot

Comment (TerribleDev) [inline scripts/points.coffee:11]: `robot.respond /give (.*) (\d*) points/i ->`

Comment (TerribleDev) [inline scripts/points.coffee:11]: respond prefixes the bot name, but in special ways

File: scripts/points.coffee [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		contestant = msg[2].toLower();
+		points = msg[3]
+		if (scoreboard.hasKey(contestant))
+			scoreboard[contestant] = scoreboard[contestant] + points
+		else
+			scoreboard[contestant] = points
+
+		msg.send "Ok, " +contestant+ " now has "+ scoreboard[contestant] + "points."
+
+	robot.hear /scoreboard/i, (msg) ->
```
Comment (TerribleDev) [inline scripts/points.coffee:22]: again you probably want `robot.respond` here

**Comment by TerribleDev** [comment]
@brianchitester  you could use gulp and mocha to unit test your scripts

**Comment by TerribleDev** [comment]
You could also inject all the environment vars, that are injected in heroku into your own shell, and just run the bot locally.

**Pull request merged**